CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Creators|creators|**REQUIRED**| The person(s) or organization(s) which contributed to the creation of the dataset. | The person(s) or organization(s) which contributed to the creation of the dataset. This can be the principal investigator, hospital, university, centre, clinic, etc. The first contact provided will be associated as the main contact for this dataset.|
+|Description|description|**REQUIRED**| A textual narrative comprised of one or more statements describing the dataset.| A short paragraph providing a rapid overview of the dataset and the context of data collection. Suggestion of items to include in the description (if applicable): main use of the dataset, population studied, study design, sample size, data collected, methods, techniques, apparatus used to generate the data.|
+|Data types|types|**REQUIRED**| Terms to describe the nature of the data. | Terms to describe the nature of the data. Data type can be single or multiple. Add a term with the [interlex URI](https://neuinfo.org/interlex/dashboard) if possible.| 
+|Version|version|**REQUIRED**| Provide the version number, or the release point of your dataset.|Provide the version number, or the release point of your dataset. Need to be a positive number. |
+|Licenses|licenses|**REQUIRED**| The use of license name abbreviations is suggested for specifying a license. Please visit [Creative Commons](https://creativecommons.org/share-your-work/) to choose the right licence for you.| The licences under which this dataset is shared.|
+|Keywords|keywords|**REQUIRED**| Tags associated with the dataset, which will help in its discovery. We suggest entering at least 5 keywords, different from the datatype.|Tags associated with the dataset, which will help in its discovery. These should be well known terms by the research community. |
+|Formats|distributions - format|**REQUIRED**| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite)Primary data files format (*example*: csv, nifti, txt, fasta).| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite).|
+|Size|distributions - size|**REQUIRED**| Total size of the dataset.| The quantity of space required on disk (or other medium) for this dataset.|
+|Units|distributions - unit|**REQUIRED**| Unit in which the size is measured.(*example*: GB)|Unit in which the size is measured.(MB -> MegaByte, GB -> GiGaByte, TB -> TeraByte and PB -> PetaByte). |
+|Access |||The information about access modality for the dataset distribution.|The information about access modality for the dataset distribution.|
+|Landing Page|distributions - access - landingPage|**REQUIRED**| Web address (URL) where the original dataset can be found.|An URL (Web page) that contains information about the associated dataset or other research object and a direct link to the object itself. |
```

Comment:
I am not sure but I think it should be written a URL. @emmetaobrien I am not 100% sure regarding that. Could you confirm?

@PapillonMcGill I am not sure I understand the object part of the text. Could it be simplified by 
```suggestion
|Landing Page|distributions - access - landingPage|**REQUIRED**| Web address (URL) where the original dataset can be found.|A URL (Web page) that contains information about the associated dataset (with a link to the dataset) or a direct link to the dataset itself. |
```

Reply:
I would favour "a URL" as well here.  And now that we have a different system in place for storing metadata about pipelines, I agree that we should cut the reference to other research objects.

Reply:
I kept "An" as URL is an abbreviation (proper grammar) even if we all say "a URL"
Other change done.

Reply:
That's not actually proper grammar, though, because the distinction in English here is more complicated than just "before a vowel"; see for example https://preply.com/en/blog/how-to-use-english-articles/#scroll-to-heading-1.

Reply:
I see, I was sure that it was always "an" before an accronym. 
@emmetaobrien Thanks for the link to explantion.